sdk/communication/Azure.Communication.Email/src/Models/EmailAttachment.cs

```diff
@@ -41,7 +41,7 @@ internal string ContentInBase64
         {
             get
             {
-                string valueToReturn = Convert.ToBase64String(Encoding.UTF8.GetBytes(Content.ToString()));
+                string valueToReturn = Convert.ToBase64String(Content.ToArray());
```

Comment:
Hi Naveen, thanks for detecting this bug and fixing it. Just wanted to check if you've tested it with other attachment types that are not pdf?

Reply:
Hi, the PR is mine.

I'm sorry, I have not really tested thoroughly  the fix, I did not have the time to clone and build the library.

Debugging through the code I verified this conversion was corrupting the data.

I've tested the new code using the previous SDK that took the base64 string already converted.

My use case was with various image types.

Reply:
Hey @davide-bergamini-sevenit, sorry for the mistake, my bad. I realized later that the PR was created by you and not Naveen. Do you have a sample pdf for which the attachment was getting corrupted to test from our side?

Reply:
nevermind, I think I have pdf that reproduces this issue now.

Reply:
I verified it with a few file types, looks good to me.